website/docs/r/nsxt_alb_pool.html.markdown

```diff
@@ -0,0 +1,202 @@
+  pool members. Can be looked up using
+  [vcd_library_certificate](/providers/vmware/vcd/latest/docs/data-sources/library_certificate) data source
+* `cn_check_enabled` - (Optional) Specifies whether to check the common name of the certificate presented by the pool
+  member
+* `domain_names` - (Optional) A set of domain names which will be used to verify the common names or subject alternative
+  names presented by the pool member certificates. It is performed only when common name check `cn_check_enabled` is
+  enabled
+* `member` - (Optional) A block to define pool members. Multiple can be used. See [Member](#member-block) and example
+  for usage details.
+* `persistence_profile` - (Optional) Persistence profile will ensure that the same user sticks to the same server for a
+  desired duration of time. If the persistence profile is unmanaged by Cloud Director, updates that leave the values
+  unchanged will continue to use the same unmanaged profile. Any changes made to the persistence profile will cause
+  Cloud Director to switch the pool to a profile managed by Cloud Director. See [Persistence
+  profile](#persistence-profile-block) and example for usage details.
+* `health_monitor` - (Optional) A block to define health monitor. Multiple can be used. See [Health
+  monitor](#health-monitor-block) and example for usage details.
+
+<a id="member-block"></a>
+## Member
+
+* `ip_address` (Required) IP address of pool member. 
+* `enabled` (Optional) defines if the pool member is enabled to receive traffic (default `true`)
+* `port` (Optional) Port for receiving traffic - overrides the root value `default_port` for individual members
+* `ratio` (Optional) Ratio of selecting eligible servers in the pool. (default `1`)
+
+### Attributes of members
+
+* `health_status` - one of `UP`, `DOWN`, `DISABLED`.
+* `detailed_health_message` - human-readable member health description. 
+* `marked_down_by` - A set of health monitors that marked the member as `DOWN` 
+
+<a id="persistence-profile-block"></a>
+## Persistence profile
+
+* `type` (Required) Type of persistence profile. One of:
+
+  * `CLIENT_IP` - The clients IP is used as the identifier and mapped to the server
+
+  * `HTTP_COOKIE` - Load Balancer inserts a cookie into HTTP responses. Cookie name must be provided as `value`
+
+  * `CUSTOM_HTTP_HEADER` - Custom, static mappings of header values to specific servers are used. Header name must be provided as `value`
+
+  * `APP_COOKIE` - Load Balancer reads existing server cookies or URI embedded data such as JSessionID. Cookie name must be provided as `value`
+
+  * `TLS` - Information is embedded in the client's SSL/TLS ticket ID. This will use default system profile System-Persistence-TLS
+
+* `value` (Optional) is required for some `type`s: ``HTTP_COOKIE`, `CUSTOM_HTTP_HEADER`, `APP_COOKIE`
+
+### Attributes of persistence profile
```

Comment:
I think the quoting has been lost at this point, as all the lines starting this one show up as blue. Looks like we have "a quote issue" in the above line, Houston :)

Reply:
Yes. Indeed it was. I think I fixed it and improved the message just a little bit.